test/tests/CookieBanner-test.js

```diff
@@ -111,4 +111,33 @@ describe('CookieBanner', () => {
     expect(_myComponent.length).toBe(1, 'cookie banner is not displaing custom child component');
   });
 
+  it('should be replaced with custom child component using function', () => {
+
+    const MyOtherComponent = React.createClass({
```

Comment:
For consistency with the rest of the cose base, please use ES6 classes for components.

In this case you can actually use a stateless component:

```js
const MyOtherComponent = ({ onAccept }) => <div className='my-other-component' onClick={this.props.onAccept} />;
```